biogeochem/EDPhysiologyMod.F90

```diff
@@ -953,11 +1030,13 @@ subroutine seed_germination( currentSite, currentPatch )
        currentPatch%seed_germination(p) =  min(currentSite%seed_bank(p) * &
              EDPftvarcon_inst%germination_timescale(p),max_germination)     
        !set the germination only under the growing season...c.xu
-       if (EDPftvarcon_inst%season_decid(p) == itrue .and. currentSite%is_cold)then 
-             currentPatch%seed_germination(p) = 0.0_r8
+       if ( (EDPftvarcon_inst%season_decid(p) == 1) .and. &
```

Comment:
Why was this and new line 1037 change from `itrue`?

Reply:
probably a merge resolution conflict, I probably updated this in a parallel pull request, but did not update this to use itrue in this PR.  it should use itrue.  And like you said earlier, should use cstatus == to different applicable conditions to avoid problems with newly added status.
perhaps we can use the "any" intrinsic function.